### PR TITLE
Add news item for group work

### PR DESCRIPTION
**Pull request by mwest1066**
Group work for shared assessments (2020-08-18) - News | PrairieLearn

### CODE REVIEW

File: news_items/018_group_work/index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  collaboratively work on an assessment together, with everyone in the
+  group submitting answers to the same questions and everyone
+  receiving the same shared scores.
+</p>
+
+<img src="student_start_assessment.png" width="697px">
+
+<p>
+  There is a flexible group assignment system that allows instructors
+  to either assign students to groups (manually or via CSV upload) or
```
Comment (mfsilva22) [inline news_items/018_group_work/index.html:25]: I think it should say `(via CSV upload or randomly generated by PrairieLearn)`

File: news_items/018_group_work/index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+</style>
+
+<p>
+  Thanks to a huge development effort led
+  by <a href="https://github.com/frankyhao">Franky Hao</a>
```
Comment (frankyhao) [inline news_items/018_group_work/index.html:13]:
```suggestion
  by <a href="https://github.com/frankyhao">Frankie Yuan</a>
```

File: news_items/018_group_work/index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  group submitting answers to the same questions and everyone
+  receiving the same shared scores.
+</p>
+
+<img src="student_start_assessment.png" width="697px">
+
+<p>
+  There is a flexible group assignment system that allows instructors
+  to either assign students to groups (via CSV upload or randomly
+  generated by PrairieLearn) or let students make their own groups.
```
Comment (frankyhao) [inline news_items/018_group_work/index.html:26]: And in the group config, instructors could set the maximum and minimum group size and students' permission (create/join/leave a group)

Comment (mwest1066) [reply news_items/018_group_work/index.html:26]: Yep, I won't describe all the details in the news items (it links to the docs for full details). This news item is just to let people know that group work exists.

**Pull request merged**